### PR TITLE
Fix popover examples when not using Portal

### PR DESCRIPTION
**Pull request by lazytype**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

**Comment by brookescarlett** [comment]
This isn't working for me when I run locally:
Screen Shot 2021-01-27 at 9 08 42 AM

**Comment by lazytype** [comment]
> This isn't working for me when I run locally:
> Screen Shot 2021-01-27 at 9 08 42 AM

Is the right border getting cut-off? what should we expect to see instead?

**Comment by brookescarlett** [comment]
Yes should expect the  border not to be cut off

File: packages/popover/src/Popover.tsx [hunk 1]
```diff
@@ -20,7 +20,6 @@ import {
 const rootPopoverStyle = css`
   position: absolute;
   transition: transform 150ms ease-in-out, opacity 150ms ease-in-out;
-  transform: translate3d(0, 0, 0) scale(1);
```
Comment (brookescarlett) [inline packages/popover/src/Popover.tsx:23]: Curious why this was causing the clipping behavior

Comment (lazytype) [reply packages/popover/src/Popover.tsx:23]: I believe because it was overriding the scale amount of 0.8 defined in positionUtils.ts

**Review by brookescarlett — APPROVED** [review]

**Comment by lazytype** [comment]
oh, guess I need a changeset now

**Pull request merged**